libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/feedback/ui/CarFeedbackOption.kt

```diff
@@ -0,0 +1,14 @@
+package com.mapbox.androidauto.car.feedback.ui
+
+import androidx.car.app.model.CarIcon
+
+data class CarFeedbackOption(
+    val title: String,
+    val icon: CarIcon,
+    val type: String? = null,
+    val subType: List<String>? = null,
```

Comment:
Wondering if `val nextPoll: CarFeedbackPoll? = null,` should replace the `subType` 🤔

Reply:
I decided to implement a more flexible approach, for example it allows you to have more than two steps. 
Also we are likely going to switch to Contribute APIs in the future, and they don't have `subType` at all.

Reply:
Can we remove subType then? I'd rather remove things we don't need, and add them when we need them.

Otherwise we'll need to deprecate unused fields

Reply:
No, it is necessary for arrival feedback at the moment. Also migration to Contribute APIs won't happen anytime soon anyway.